packages/react-cosmos-webpack/src/export.js

```diff
@@ -0,0 +1,58 @@
+import path from 'path';
+import webpack from 'webpack';
+import { argv } from 'yargs';
+import fs from 'fs-extra';
+import getWebpackConfig from './webpack-config';
+import getDefaultWebpackConfig from './default-webpack-config';
+import importModule from 'react-cosmos-utils/lib/import-module';
+import getCosmosConfig from 'react-cosmos-config';
+
+process.env.NODE_ENV = 'production';
+
+const moduleExists = modulePath => {
+  try {
+    return modulePath && require.resolve(modulePath) && true;
+  } catch (err) {
+    return false;
+  }
+};
+
+module.exports = function startExport() {
+  const cosmosConfigPath = argv.config;
+  const cosmosConfig = getCosmosConfig(cosmosConfigPath);
+
+  const {
+    webpackConfigPath
+  } = cosmosConfig;
+
+  let userWebpackConfig;
+  if (moduleExists(webpackConfigPath)) {
+    console.log(`[Cosmos Export] Using webpack config found at ${webpackConfigPath}`);
+    userWebpackConfig = importModule(require(webpackConfigPath));
+  } else {
+    console.log('[Cosmos Export] No webpack config found, using default configuration');
+    userWebpackConfig = getDefaultWebpackConfig();
+  }
+
+  const cosmosWebpackConfig = getWebpackConfig(userWebpackConfig, cosmosConfigPath, true);
+  const compiler = webpack(cosmosWebpackConfig);
+
+  compiler.run((err, stats) => {
+    if (err) {
+      console.error('Export Failed! See error below:');
+      console.error(err);
+    } else {
+      console.log(stats);
+      try {
+        fs.copySync(path.join(__dirname, 'static/favicon.ico'), `${cosmosWebpackConfig.output.path}/favicon.ico`);
```

Comment:
This is a bit more tricky. Cosmos uses two index.htmls:
- [Index of the playground](https://github.com/react-cosmos/react-cosmos/blob/ed8ef3e09645cd69d0086339ea6e169294bd4aef/packages/react-cosmos-webpack/src/static/index.html), used to render nav and controls (this one is missing from the export at the moment)
- Index of the user, used for the contents of the loader frame (this one is generated by the export, but should be placed in the `loader/` dir, next to the `bundle.js` file)